modules/nw-multus-tap-object.adoc

```diff
@@ -81,7 +81,66 @@ The following example configures an additional network named `mynet`:
 }
 ----
 
+[id="nw-multus-enable-container_use_devices_{context}"]
+
+== Setting SELinux boolean for the TAP CNI plugin
+
+To create the tap device with the `container_t` SELinux context, enable the `container_use_devices` boolean on the host by using the Machine Config Operator (MCO).
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a new YAML file named for example `setsebool-container-use-devices.yaml` with the following details:
++
+[source, yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-worker-setsebool
+spec:
+ config:
+ignition:
+version: 3.2.0
+systemd:
+units:
+- contents: |
+    [Unit]
+    Description=Set SELinux booleans
+    Before=kubelet.service
+
+    [Service]
+    Type=oneshot
+    ExecStart=/usr/sbin/setsebool container_use_devices=on
+    RemainAfterExit=true
+
+    [Install]
+    WantedBy=multi-user.target graphical.target
+   enabled: true
+   name: setsebool.service
+----
++
+
+. Create the new `MachineConfig` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f setsebool-container-use-devices.yaml
+----
++
 [NOTE]
 ====
-To create the tap device with the `container_t` SELinux context, enable the `container_use_devices` boolean on the host by using the Machine Config Operator (MCO).
-====
\ No newline at end of file
+Applying any changes to the `MachineConfig` object causes all affected nodes to gracefully reboot after the change is applied. This update can take some time to be applied.
+====
++
+. Verify the change is applied by running the following command:
++
+[source,terminal]
+----
+$ oc get machineconfigpools
```

Comment:
Please add expected output with a comment that all nodes have to be in Updated and Ready state Something like:
```
NAME        CONFIG                                                UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
master      rendered-master-e5e0c8e8be9194e7c5a882e047379cfa      True      False      False      3              3                   3                     0                      7d2h
worker   rendered-workercnf-d6c9ca107fba6cd76cdcbfcedcafa0f2   False     True       False      3              3                   3                     0                      7d
```